### PR TITLE
Task 10

### PR DESCRIPTION
**Pull request by GualbertoVillarroel**

### CODE REVIEW

File: Gualberto/Ruby/Day9/Practice4.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+	def print_hashValue hash
+		puts"The values are: #{hash.values}"
+	end
+
+	def print_hash hash
+		puts"The hash is: #{hash}"
+	end
+
+	def isAKey hash
```
Comment (karimeSalomon) [inline Gualberto/Ruby/Day9/Practice4.rb]: Like this is a method that will give you true or false you should define it as a predictive method I mean isAKey?

File: Gualberto/Ruby/Day9/Practice4.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		print"Insert a key to search in the hash: "
+		key = gets.chomp.to_s
+		puts hash.has_key?(key)
+
+	end
+	def isAValue hash
```
Comment (karimeSalomon) [inline Gualberto/Ruby/Day9/Practice4.rb]: The same previous comment

File: Gualberto/Ruby/Day9/Practice4.rb (hunk 1 above)
Comment (GualbertoVillarroel) [inline Gualberto/Ruby/Day9/Practice4.rb]: Ok, thanks.

File: Gualberto/Ruby/Day9/Practice4.rb (hunk 2 above)
Comment (GualbertoVillarroel) [inline Gualberto/Ruby/Day9/Practice4.rb]: Ok

**Pull request merged**